go/tools/builders/compilepkg.go

```diff
@@ -82,6 +83,7 @@ func compilePkg(args []string) error {
 	fs.StringVar(&cgoExportHPath, "cgoexport", "", "The _cgo_exports.h file to write")
 	fs.StringVar(&testFilter, "testfilter", "off", "Controls test package filtering")
 	fs.StringVar(&coverFormat, "cover_format", "", "Emit source file paths in coverage instrumentation suitable for the specified coverage format")
+	fs.Var(&recompileInternalDeps, "recompile_internal_deps", "Import path, package path, and file name of a direct dependency that needs to be recompiled, separated by '='.")
```

Comment:
Looks like you only need import path to check the cycles. Can you only pass the import path? It makes the command line shorter.